### PR TITLE
Updated Harness Logic to Ox_Inventory

### PR DESCRIPTION
**Pull request by xViperAG**
**Describe Pull request**
First, make sure you've read and are following the contribution guidelines and style guide and your code reflects that.
Write up a clear and concise description of what your pull request adds or fixes and if it's an added feature explain why you think it should be included in the core.

List of changes:

- Progressbar changes to ox_lib
- Logic changed from qb-inventory itemData to ox_inventory metadata
- Preview of uses left in metadata

If your PR is to fix an issue mention that issue here

**Questions (please complete the following information):**
- Have you personally loaded this code into an updated qbcore project and checked all it's functionality? [yes/no] (Be honest) Yes, I have checked that all that is seen below works.
- Does your code fit the style guidelines? [yes/no] Yes.
- Does your PR fit the contribution guidelines? [yes/no] Yes

### CODE REVIEW

File: server/main.lua [hunk 1]
```diff
@@ -23,29 +23,30 @@ RegisterNetEvent('equip:harness', function(item)
 
     if not Player then return end
 
-    if not Player.PlayerData.items[item.slot].info.uses then
-        Player.PlayerData.items[item.slot].info.uses = 19
-        Player.Functions.SetInventory(Player.PlayerData.items)
-    elseif Player.PlayerData.items[item.slot].info.uses == 1 then
-        TriggerClientEvent('inventory:client:ItemBox', src, QBCore.Shared.Items['harness'], 'remove')
-        Player.Functions.RemoveItem('harness', 1)
+    if not item.metadata.harnessuses == nil then
```
Comment (Manason) [inline server/main.lua]:
```suggestion
    if item.metadata.harnessuses then
```

File: server/main.lua [hunk 2]
```diff
@@ -23,29 +23,30 @@ RegisterNetEvent('equip:harness', function(item)
...
+        item.metadata.harnessuses = 20
+        exports.ox_inventory:SetMetadata(src, item.slot, item.metadata)
+    elseif item.metadata.harnessuses == 1 then
```
Comment (Manason) [inline server/main.lua:29]: this will never be true

**Review by Manason — CHANGES_REQUESTED** [review]

**Comment by xViperAG** [comment]
The reason that the '== nil' was left in the code was because without it, it would return a nil error, so this is a fix for that to correct the nil error in the harness.

**Comment by YvanCywan** [comment]
> The reason that the '== nil' was left in the code was because without it, it would return a nil error, so this is a fix for that to correct the nil error in the harness.

Nil always returns false when in an if statement, so doing
```lua
if not nilValue then
   body
end
```
and doing
```lua
if nilValue == nil then
   body
end
``` 
Does the same thing

**Pull request merged**